internal/build/job/maker.go

```diff
@@ -19,7 +19,8 @@ import (
 //go:generate mockgen -source=maker.go -package=job -destination=mock_maker.go
 
 type Maker interface {
-	MakeJobTemplate(mod kmmv1beta1.Module, buildConfig *kmmv1beta1.Build, targetKernel, containerImage string, pushImage bool) (*batchv1.Job, error)
+	MakeJobTemplate(mod kmmv1beta1.Module, buildConfig *kmmv1beta1.Build, targetKernel, containerImage string,
```

Comment:
If we TLS option fo pushing will be defined in the Build , and not in kernelMapping, then this API needs not to be changed

Reply:
It is not. It is defined in the kernelMapping/moduleLoader